tests/app-test.js

```diff
@@ -1,10 +1,77 @@
-/* global describe, beforeEach, afterEach, it */
+/* global describe, beforeEach, afterEach */
 import { expect } from 'chai';
+import $ from 'jquery';
+import sinon from 'sinon';
+import it from './it-will';
```

Comment:
i like the`it.immediately` route, but i think we should make a note about it in the readme.  people see `it` and think `it` - even though we're accepting this as our default it's not most people's default